pure.zsh

```diff
@@ -83,7 +83,7 @@ prompt_pure_precmd() {
 		# make sure working tree is not $HOME
 		[[ "$(command git rev-parse --show-toplevel)" != "$HOME" ]] &&
 		# check check if there is anything to pull
-		command git fetch &>/dev/null &&
+		command git -c gc.auto=0 fetch &>/dev/null &&
```

Comment:
Can you use the long-flag instead of `-c`?

And should every `git` command have this?

@mafredri